### ISSUE TITLE
Training MGAN?

### ISSUE THREAD

**Issue by ShaneGilroy** [hasanirtiza/Pedestron#50]
Is it possible to train MGAN using Pedestron?

**Comment by hasanirtiza** [comment]
In theory, it should be. However, currently we have not added the script but it should be doable. As [official MGAN](https://github.com/Leotju/MGAN) repo does not provide configs for the training part. Based on the paper, my impression is that they are quite similar/ or exactly are that of [Adaptive Faster R-CNN](https://arxiv.org/pdf/1702.05693.pdf), of course they use the visible box/mask additionally.

**Comment by ShaneGilroy** [comment]
Great thanks.

**Comment by yutongy98** [comment]
Hi, I also found that the config file of the MGAN doesn't provide the training script until now, if it means until now we still cannot train the MGAN using Pedestron? @hasanirtiza

**Comment by hasanirtiza** [comment]
True, unfortunately the authors never released the training configs for MGAN. You can check on the [official repo](https://github.com/Leotju/MGAN). However, I did stumble upon this [config](https://github.com/ShaneGilroy/Mask-guided-attention-network/blob/master/configs/mgan.py), it does provide training configs, I have never tried it myself, may be you can see if it works.

**Comment by yutongy98** [comment]
Thanks! Because I saw the pre-trained MAGN model in the benchmark table listed in the README is available, but the config for the training MGAN doesn't exist so I was little confused. How did you generate the pre-trained MGAN model?
